server/server.go

```diff
@@ -360,6 +360,9 @@ type Server struct {
 
 	// Queue to process JS API requests that come from routes (or gateways)
 	jsAPIRoutedReqs *ipQueue[*jsAPIRoutedReq]
+
+	// Whether moving NRG traffic into accounts is permitted on this server.
+	accountNRGAllowed atomic.Bool
```

Comment:
This will always be true yes for 2.11 servers?

Reply:
Yes, it's mostly there to test mixed compatibility in the unit tests.

Reply:
ok, still not a fan of things in released server that are only for tests..